core/src/main/scala/chandu0101/scalajs/react/components/ReactTable.scala

```diff
@@ -126,35 +120,30 @@ object ReactTable {
           S.filteredModels.length,
           S.offset,
           onNextClick,
-          onPreviousClick
-        )
-      )
+          onPreviousClick))
   }
 
-  def getIntSort(key: String) = (m1: Model, m2: Model) => m1(key).asInstanceOf[Int] < m2(key).asInstanceOf[Int]
+  def getIntSort(key: String) = (m1: Model, m2: Model) ⇒ m1(key).asInstanceOf[Int] < m2(key).asInstanceOf[Int]
```

Comment:
I took out all the unicode arrows (even though i like them myself) because people complained about them, and in any case we should be consistent, so we dont pollute diffs for all future. can we go with `=>` for now?